### PR TITLE
Add Python docker app

### PR DESCRIPTION
**Pull request by jgroom33**

### CODE REVIEW

File: theia-python-docker/next.package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        "@theia/messages": "next",
+        "@theia/navigator": "next",
+        "@theia/terminal": "next",
+        "@theia/outline-view": "next",
+        "@theia/preferences": "next",
+        "@theia/monaco": "next",
+        "@theia/git": "next",
+        "@theia/file-search": "next",
+        "@theia/markers": "next",
+        "@theia/textmate-grammars": "latest",
```
Comment (akosyakov) [inline theia-python-docker/next.package.json]: `next`

Comment (jgroom33) [reply theia-python-docker/next.package.json]: thanks.  good catch

**Comment by marcdumais-work** [comment]
@jgroom33 Can you please rebase your PR once https://github.com/theia-ide/theia-apps/pull/69 is merged? That should fix the Travis errors.

**Comment by jgroom33** [comment]
sure

**Comment by marcdumais-work** [comment]
@jgroom33 NVM, #69 will not help.

**Comment by jgroom33** [comment]
I think #69 is the solution to most problems

**Comment by marcdumais-work** [comment]
That's what I thought too, but it turns out that we can't make the GH token work for Travis, when the PR comes from a fork. So it would not help, e.g. for this PR here, I think:

https://docs.travis-ci.com/user/pull-requests/#pull-requests-and-security-restrictions

**Pull request merged**